contributing/documentation/standards.rst

```diff
@@ -57,6 +57,8 @@ Code Examples
   Unless the example requires a custom bundle, make sure to always use the
   ``AppBundle`` bundle to store your code;
 * Use ``Acme`` when the code requires a vendor name;
+* Use ``example.com`` as the domain of sample URLs and ``example.org`` when a
+  second domain is required. Both domains are `reserved by the IANA`_.
```

Comment:
I wouldn't add an order in the standard, just what domains you can use: example.com, example.org and example.net

I would also link to the RFC instead: http://tools.ietf.org/html/rfc2606#section-3